src/main/java/pyke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,57 @@
+package pyke.command;
+
+import pyke.exception.EmptyDescriptionException;
+import pyke.exception.EmptyTimeException;
+import pyke.exception.PykeException;
+import pyke.task.Deadline;
+import pyke.ui.Ui;
+import pyke.util.TaskList;
+import pyke.util.Storage;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class AddDeadlineCommand extends Command{
```

Comment:
Code looks good! However, there seems to be some inconsistency with the spacing (for example there should be a white space between "Command" and "{")